artifacts/definitions/Windows/Detection/TemplateInjection.yaml

```diff
@@ -0,0 +1,108 @@
+name: Windows.Detection.TemplateInjection
+description: |
+    This content will detect injected templates in Office documents.   
+    
+    Template injection is a form of defence evasion where a malicious macro 
+    is loaded into an OOXML document via a resource file masquerading as 
+    an office template.   
+
+author: Matt Green - @mgreen27
+
+reference:
+  - https://attack.mitre.org/techniques/T1221/
+  - https://www.sans.org/reading-room/whitepapers/testing/template-injection-attacks-bypassing-security-controls-living-land-38780
+
+parameters:
+  - name: SearchGlob
+    description: Glob to search
+    default: C:\Users\**\*.{docx,dotx,xlsx,xltx,pptx,potx,ppsx,sldx}
+  - name: TemplateRegex
+    description: Regex to search inside resource section.
+    default: '^(https?|smb|\\\\\\\\)'
+  - name: UploadDocument
+    type: bool
+    description: Select to upload document on detection.
+  
+sources:
+  - precondition: SELECT OS From info() where OS = 'windows'
+  
+    queries: 
+     - |
+      -- Find target docs
+      LET office_docs = SELECT  *
+            FullPath AS OfficePath,
+            timestamp(epoch=Mtime.Sec) as OfficeMtime,
+            Size as OfficeSize,
+            url(
```

Comment:
Not used?

Reply:
removed the unused variables and also changed the default output to align with standard output. Only "Section" details retain their column names. FullPath,Hash,Mtime,Size
![image](https://user-images.githubusercontent.com/13081800/100533911-df062000-325d-11eb-967c-0ad8334e3752.png)